tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -699,6 +709,21 @@ def supported_property_hash(tests, id, property_hash)
     # because :resource hash currently uses strings for keys
     copy.delete(prop_symbol.to_s)
   end
+  return copy if version_unsupported_properties(tests, id).empty?
+  lim = full_version.split[0].tr('(', '.').tr(')', '.').chomp('.')
+  # due to a bug in Gem::Version, we need to append a letter
+  # to the version field if the to be compared version
+  # has a letter at the end
```

Comment:
Same comment as the NU side here.  Please add an example of real versions in the comments for this.